requirements.txt

```diff
@@ -1,2 +1,2 @@
 github3.py == 1.3.0
-python-jenkins>=0.4.13
+python-jenkins==0.4.13
```

Comment:
My general thought here would be to upgrade the library and related script with the new payload structure, curious if there is a reason we decided intead to use an older version of the library ?

Reply:
The library is responsible for composing the request inside the [set_next_build_number](https://opendev.org/jjb/python-jenkins/src/commit/6c2e3be09a78d1b5f9be894188b5b347086bd078/jenkins/__init__.py#L1218) method, and in this case, because of the way the requirement was set, `pip` was installing the latest version `1.8.2`

Reply:
cool sounds good to me.